test/uk/gov/hmrc/cdsreimbursementclaim/controllers/SubmitClaimControllerSpec.scala

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.cdsreimbursementclaim.controllers
+
+import play.api.http.{HeaderNames, Status}
+import play.api.libs.json.{JsObject, JsString, JsValue, Json, Writes}
+import play.api.test.Helpers._
+import play.api.test._
+import uk.gov.hmrc.cdsreimbursementclaim.config.AppConfig
+import uk.gov.hmrc.cdsreimbursementclaim.services.EisService
+import uk.gov.hmrc.http.{HeaderCarrier, HttpClient, HttpReads, HttpResponse}
+
+import scala.concurrent.{ExecutionContext, Future}
+
+class SubmitClaimControllerSpec extends ControllerSpec with DefaultAwaitTimeout {
+
+  val httpClient = mock[HttpClient]
+  val appConfig  = instanceOf[AppConfig]
+  val eisService = new EisService(appConfig, httpClient)
```

Comment:
if you define a trait as I mentioned above then you can use Scalamock to mock out the call instead of instantiating the actual class here.